hq/app/model/models.scala

```diff
@@ -288,5 +288,5 @@ case class IamAuditUser(id: String, awsAccount: String, username: String, alerts
 object IamAuditUser {
   implicit val iamAuditUserWrites = Json.writes[IamAuditUser]
 }
-case class IamNotification(iamUser: IamAuditUser, anghammaradNotification: Notification)
+case class IamNotification(warningN: Option[Notification], finalN: Option[Notification], alertedUsers: Seq[IamAuditUser])
```

Comment:
I would rename warningN -> warning as the 'notification' bit should be obvious from the type

Reply:
I was thinking the same, but I get a keyword clash for `final`, so I thought I would keep them both the same.

Reply:
👍🏻